api/utilities/response_generation.py

```
@@ -0,0 +1,445 @@
+            pad_token_id=tokenizer.eos_token_id,
+            eos_token_id=tokenizer.eos_token_id
+        )
+
+        # Decode and extract response
+        full_output = tokenizer.decode(outputs[0], skip_special_tokens=True).strip()
+
+        print("FULL RAW RESPONSE:", full_output)  # DEBUG
+        print("Input length:", inputs.input_ids.shape[1])
+        return full_output
+
+    except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)